test/unit/classes/galery/mandant/MandantTest.php

```diff
@@ -0,0 +1,15 @@
+<?php
+/**
```

Comment:
Please remove this unnecessary comment. We have this information in our source control system